endpoints/openrtb2/sample-requests/invalid-whole/user-ext-eids-uids-id-empty.json

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
Instead of removing these tests cases, please modify them to prove the new behavior.